.github/workflows/test.yml

```diff
@@ -125,7 +125,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       tendermint:
-        image: informaldev/tendermint:0.34.0
+        image: informaldev/tendermint:0.34.13
```

Comment:
Important to note here that the `block_search` endpoint was only introduced in [Tendermint v0.34.9](https://github.com/tendermint/tendermint/blob/master/CHANGELOG.md#v0349).